### DISCUSSION TITLE
Can't login

### DISCUSSION THREAD

**Discussion by Captain-Barge** [nesbox/TIC-80#2402] [Q&A]
I tried to login the tic-80 website to purchase the PRO version (it's been about a year since my last login) but my login credentials don't seem to work. A password reset seems to work at first & I can reset my password, but once it's completed and I try to login again with the new password it rejects my credentials.

I've tried LibreWolf and Chromium browsers on Ubuntu 22.04. Not using a VPN. My login email is a protonmail account (proton.me to be exact).

Now I'm worried that even if I manage to get in and buy the PRO version I might lose it just due to my account "expiring". Any advice here?

**Comment by Skeptim** [comment]
It looks like it's the same issue than #2375, explain your problem there to help nesbox solve it.
However, the PRO version cannot be purchased from tic80.com but from https://nesbox.itch.io/tic80

**Reply by Captain-Barge** [reply]
Ah, thanks a ton! I'll check the itch.io page!
